playbooks/reboot_cluster.yml

```diff
@@ -11,6 +11,8 @@
   - name: waiting for server to come back
     wait_for: host={{ inventory_hostname }}
               state=started
+              connect_timeout=900
```

Comment:
Can explain how connect_timeout and timeout works. What will we see on reboot with this changes?